### ISSUE TITLE
National Map v2018-04-16 pre-release test - Duplicate layers in the Catalogue!!

### ISSUE THREAD

**Issue by EhsanCanberra** [TerriaJS/nationalmap#740]
**Multiple duplicate layer names in Catalogue entry!!** 

Please review the whole Catalogue group: 

Data group: Data Providers/Geoscience Australia/*

**Comment by maxious** [comment]
Indeed. That dataset has had 2 WMS links since it was created 2017-06-25 based on the XML https://ecat.ga.gov.au/geonetwork/srv/api/records/2b741757-065f-46c6-e053-12a3070a5ed6/formatters/xml where it appears as 2 online resources under MD_DigitalTransferOptions and connectPoint

**Comment by AnaBelgun** [comment]
Thanks @maxious . Does GA need to clean up its resource entry or data.gov.au has to do that?

**Comment by AnaBelgun** [comment]
Awesome. Thanks @maxious!
